pkg/bundle/setup/clustersetup.sh

```
@@ -92,16 +89,16 @@ wait_for_resource() {
 #Replaces the default pubkey with the new one just generated to avoid the mysterious service to replace it later on :-\
 replace_default_pubkey() {
     pr_info "Updating the public key resource for machine config operator"
-    local pub_key=$(tr -d '\n\r' < /home/core/id_rsa.pub)
+    local pub_key=$(tr -d '\n\r' </home/core/id_rsa.pub)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)